bokehjs/src/coffee/models/annotations/label.coffee

```diff
@@ -102,8 +103,8 @@ class LabelView extends Renderer.View
     for i in [0...@_text.length]
       ctx.save()
 
-      ctx.rotate(@mget('angle'))
       ctx.translate(@sx[i] + @_x_offset[i], @sy[i] - @_y_offset[i])
+      ctx.rotate(@_angle[i])
```

Comment:
Definitely good to move this after the translate. I'm pretty sure I'll be using translate for my side panels work, so this makes a lot more sense.